app/controllers/concerns/media_resources/crud_actions.rb

```diff
@@ -21,6 +21,18 @@ def list_meta_data
       def index
         resources = auth_policy_scope(current_user, model_klass)
         @get = presenterify(resources, nil)
+
+        if @get.resources.empty?
+          if controller_name == 'media_entries'
+            collections = auth_policy_scope(current_user, Collection)
+            collections_get = presenterify(
+              collections, Presenters::Collections::Collections)
+            unless collections_get.resources.empty?
+              @get.try_collections = true
+            end
+          end
+        end
```

Comment:
Das gehört schon eher in einen controller, nur in dem Concern hat es nichts verloren. Einfach die ganze index methode in `media_entries` controllers (dann fällt auch ein if weg) und gut ists.